src/types.h

```diff
@@ -57,6 +57,9 @@ struct Converter {
      * 'Source' may be given relative to 'relativeTo', e.g. "toggle" for booleans.
      */
     static T parse(const std::string& source, T const* relativeTo);
+    static T parseDefault(const std::string& source, const T& relativeTo) {
```

Comment:
:rabbit: The naming of this method confuses me. First I thought it indicates that the second value should be used as a default when parsing fails. But then I realized it is not used like that. What does `default` stand for here? Or could this method also just share the name with existing `parse()`?

Reply:
It can't be called `parse()` because then the call to `Converter<bool>::parse()` is ambigous. What do you think about the names `parseWithDefault` or `parseRelativeTo`, or something else?